### ISSUE TITLE
Stop using @-moz-document in CSS

### ISSUE THREAD

**Issue by webknjaz** [rtfd/sphinx_rtd_theme#640]
### Problem

Basically this theme tables rendering is broken in Firefox nightly (`v61.0b3`), but as I understand it might affect stable users over time.
This issue follows up on https://github.com/ansible/ansible/issues/40007#issuecomment-389194172 reported by @2vek.
@emilio has pointed out that `@-moz-document` should not be used.
Ref: https://bugzilla.mozilla.org/show_bug.cgi?id=1461687

### Reproducible Project

http://docs.ansible.com/ansible/latest/modules/copy_module.html

#### Error Logs/Results

![ansible copy module docs](https://user-images.githubusercontent.com/2778313/39925207-5876013a-5548-11e8-8cfe-a1f117ace8c7.png)

#### Expected Results

It should not break tables rendering in browser.

### Environment Info

- Python Version: —
- Sphinx Version: —
- RTD Theme Version: —

**Comment by Blendify** [comment]
I dont think this is an issue in the theme but in your own css, try updating your css file with our latest.
